providers/capa/standard/test_data/cluster_values.yaml

```diff
@@ -1,23 +1,24 @@
-metadata:
-  name: "{{ .ClusterName }}"
-  description: "E2E Test cluster"
-  organization: "{{ .Organization }}"
+global:
```

Comment:
For now fixing the standard test. Once there is a new cluster-aws release, we'll fix also the upgrade test.